htdocs/commande/list.php

```diff
@@ -187,6 +187,7 @@
 	'c.multicurrency_total_vat'=>array('label'=>'MulticurrencyAmountVAT', 'checked'=>0, 'enabled'=>(empty($conf->multicurrency->enabled) ? 0 : 1), 'position'=>105),
 	'c.multicurrency_total_ttc'=>array('label'=>'MulticurrencyAmountTTC', 'checked'=>0, 'enabled'=>(empty($conf->multicurrency->enabled) ? 0 : 1), 'position'=>110),
 	'u.login'=>array('label'=>"Author", 'checked'=>1, 'position'=>115),
+	'sale_representative'=>array('label'=>"SaleRepresentativesOfThirdParty", 'position'=>116),
```

Comment:
Add also 'checked'=>0 so the field is not enabled by default.

Reply:
@eldy it's done.